wvd-sh/arm-template-customization/readme.md

```diff
@@ -0,0 +1,21 @@
+# Azure Virtual Desktop - Post Update Custom Configuration solution
```

Comment:
We should change the name to custom configuration

Reply:
@dhrump the feature in the portal is called "Post Update Custom Configuration".  I believe it makes sense to at least keep that text so customers can easily identify the solution.

Reply:
We are changing name internally. I will update the changes and push our the PR